hudl_behave_teamcity/__init__.py

```diff
@@ -78,11 +78,16 @@ def result(self, step_result):
         if self.current_scenario.status == "untested":
             return
 
-        if self.current_scenario.status == "passed":
+        status = self.current_scenario.status
+        if type(self.current_scenario.status) is not str:
```

Comment:
Can be `if type(status) is not str`, or you could use the ternary operator

Reply:
Good call. I spend more time on PR descriptions than reviewing my own code sometimes.

I'm not a huge fan of the ternary operator only because it's just a little harder to understand at a glance.